packages/web-component-tester/test/retry-failures-reporter.ts

```diff
@@ -78,7 +80,7 @@ module.exports = class RetryFailuresReporter extends Mocha.reporters.Spec {
           fails.map((t) => ' - ' + t).join('\n')));
 
       const retryResults = spawnSync(process.argv0, args, {
-        cwd: process.cwd(),
+        cwd: initialCurrentWorkingDirectory,
```

Comment:
Ensures we use the cwd() at the beginning of the run instead of whatever it might be when we are done.